### PR TITLE
[zh-cn] Update kubelet-authn-authz.md

### PR DESCRIPTION
**Pull request by Sea-n**
Change `/zh/docs/` to `/zh-cn/docs/` and resync some small nits.

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md [hunk 1]
```diff
@@ -1,99 +1,99 @@
...
 要对 kubelet 的 HTTPS 端点启用 X509 客户端证书认证：
 
-<!--  
+<!--
 * start the kubelet with the `--client-ca-file` flag, providing a CA bundle to verify client certificates with
 * start the apiserver with `--kubelet-client-certificate` and `--kubelet-client-key` flags
 * see the [apiserver authentication documentation](/docs/reference/access-authn-authz/authentication/#x509-client-certs) for more details
 -->
 * 带 `--client-ca-file` 标志启动 kubelet，提供一个 CA 证书包以供验证客户端证书
 * 带 `--kubelet-client-certificate` 和 `--kubelet-client-key` 标志启动 apiserver
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md:62]:
```suggestion
* 带 `--kubelet-client-certificate` 和 `--kubelet-client-key` 标志启动 API 服务器
```

File: content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
 * 有关更多详细信息，请参见
-  [apiserver 身份验证文档](/zh/docs/reference/access-authn-authz/authentication/#x509-client-certs)
+  [apiserver 身份验证文档](/zh-cn/docs/reference/access-authn-authz/authentication/#x509-client-certs)
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md:64]:
```suggestion
  [API 服务器身份验证文档](/zh-cn/docs/reference/access-authn-authz/authentication/#x509-client-certs)
```
ref:
https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7

File: content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md [hunk 3]
```diff
@@ -117,19 +117,19 @@ To subdivide access to the kubelet API, delegate authorization to the API server
 * kubelet 调用已配置的 API 服务器上的 `SubjectAccessReview` API，
   以确定每个请求是否得到鉴权
 
-<!--  
+<!--
 The kubelet authorizes API requests using the same [request attributes](/docs/reference/access-authn-authz/authorization/#review-your-request-attributes) approach as the apiserver.
 -->
 kubelet 使用与 apiserver 相同的
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz.md:123]:
```suggestion
kubelet 使用与 API 服务器相同的
```

**Comment by Sea-n** [comment]
I'm not sure should we translate `apiserver` to `API 服务器` or not.
I would prefer to leave it as is and wait for others' opinion.

**Comment by ydFu** [comment]
> I'm not sure should we translate `apiserver` to `API 服务器` or not. I would prefer to leave it as is and wait for others' opinion.

This section lists common terminology for the unification.
https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7

**Comment by tengqm** [comment]
> I'm not sure should we translate `apiserver` to `API 服务器` or not.
> I would prefer to leave it as is and wait for others' opinion.

This was mentioned in the English style guide. Use 'API Server' rather than 'apiserver'. See [here](https://kubernetes.io/docs/contribute/style/style-guide/#use-a-general-descriptor-over-a-component-name).

**Pull request merged**